.ci/validation.sh

```diff
@@ -1076,6 +1076,11 @@ git-check-pull-number)
   done
   ;;
 
+assembly-site)
+  mvn -e --no-transfer-progress package -Passembly
+  mvn -e --no-transfer-progress site -Dlinkcheck.skip=true
+  ;;
```

Comment:
Moved this out of drone file to it's own script.

Reply:
Can we split them different tasks of validation.sh ?

Reply:
Please allow execution not such mvn commands separately